src/grammar.pegjs

```diff
@@ -504,7 +504,7 @@ htmlTag = tag:$('<' '/'? [a-z]+ [^>]* '>') {
   });
 }
 
-reference = '{' !('++'/'--') _ ref:(call / value / token)? _ close:'}'? {
+reference = '{' !('++'/'--') _ ref:(call / token / value)? _ close:'}'? {
```

Comment:
This flip is probably what caused the unexpected changes you're seeing in tests. PEG try to parse in order and there's some overlap between these two rules

Reply:
Yep. And the reason for flipping it was because it started parsing the closing parenthesis as part of the token.